src/llama-model.h

```diff
@@ -402,7 +402,10 @@ struct llama_model {
 
     // note: can mutate `cparams`
     // TODO: move this to new llm_arch_model_i interface
-    llama_memory_i * create_memory(const llama_memory_params & params, llama_cparams & cparams) const;
+    llama_memory_i * create_memory(
```

Comment:
I just realized that this is entirely unnecessary since `llama_model` already has `hparams` as a member variable 🤦